### PR TITLE
Update elixir version

### PR DESCRIPTION
**Pull request by dmitrytrager**

### CODE REVIEW

**Comment by dmitrytrager** [comment]
@AlexeyKuznetzov could you please help with code style checks in bin/ci?

**Comment by ghost** [comment]
@dmitrytrager It looks like the new version of `Credo` forces setting `spec` types for functions. It's controversial proposition and should be discussed.

File: lib/cli.ex [hunk 1]
```diff
@@ -3,6 +3,7 @@ defmodule ElixirBase.CLI do
     CLI entry point
   """
 
+  @spec main(any) :: {:od, pid}
```
Comment (ghost) [inline lib/cli.ex]: :ok

**Comment by dmitrytrager** [comment]
@AlexeyKuznetzov done

**Review by ghost — APPROVED** [review]

**Pull request merged**